test/Experimental/JuLie-test.jl

```diff
@@ -1,3 +1,4 @@
 include("JuLie/tableaux.jl")
 include("JuLie/schur_polynomials.jl")
 include("JuLie/partitions.jl")
+include("JuLie/quantum_analogs.jl")
```

Comment:
please always end files with a trailing new line (and ideally configure your editor to do that automatically, e.g. by making it honor our `.editoconfig` file, as described in the dev docs -- assuming you are using an editor with that capability)
```suggestion
include("JuLie/quantum_analogs.jl")

```